.github/workflows/developments_build.yml

```diff
@@ -46,8 +55,14 @@ jobs:
         working-directory: ./
         run: ./bash/build_env_setup.sh
 
-      - name: 1. dataloading for EDM builds
-        run: ./devdb.sh dataloading edm && ls -l
+      - name: Set product environment variables
+        run: cat version.env >> "$GITHUB_ENV"
```

Comment:
Would like to change this a bit but that's what I'm already doing in my branch 🙂